docs/web3j_openapi.md

```diff
@@ -0,0 +1,493 @@
+
+## Generate a project:
+To generate an OpenAPI project using the [Web3j-OpenAPI](https://github.com/web3j/web3j-openapi) generator, you need to have the [Epirus-CLI](https://docs.epirus.io/quickstart) installed on your machine (Note - the Epirus CLI has replaced the Web3j CLI). 
+It’s easy to do (for Windows instructions head [here](https://docs.epirus.io/quickstart/#installation)):
+	
+```
+$ curl -L get.epirus.io | sh
+```
+
+Then, you can run the
+```
+$ epirus openapi import --help
+```
+To check the generator available options. 
+
+![image](img/Web3j-OpenAPI/Generator_help.png)
+
+In this case, we will be using the above Hello World contract.
+
+Put that contract in a file named `HelloWorld.sol` for starters.
+
+Then, execute the following command: 
+```	
+$ epirus openapi import  \
+    -s HelloWorld.sol \
+    --package-name com.tutorial \
+    --name HelloWorldProject \
+    --output . 
+```
+You should be seeing logs similar to the following:
+![image](img/Web3j-OpenAPI/Generator_logs.png)
+
+## Configure the project
+
+You can run the project easily, without specify any configuration, using the [Epirus-cli](https://docs.epirus.io/quickstart/#deployment).
+
+If not, you need to specify runtime parameters. for example, the `private key` or `wallet file` for the signing,
+the `node endpoint` to connect to, etc.
+
+To see the available options, try to run the `distributions` or the `JAR` with the `--help` flag. You'll get the following display:
+
+![image](img/Web3j-OpenAPI/Server_help.png)
+
+To specify these parameters, you can set the following environment variables:
+
+```shell script
+$ export WEB3J_ENDPOINT=<link to your Ethereum node>
+$ export WEB3J_PRIVATE_KEY=<your private key>
+$ export WEB3J_OPENAPI_HOST=localhost
+$ export WEB3J_OPENAPI_PORT=9090
+```
+
+For more information, check the configuration section below.
+
+## Run the project
+We can run the project directly : 
+```
+$ ./gradlew run
+```
+
+You should be able to run the server and see the following:
+
+![image](img/Web3j-OpenAPI/Server_logs.png)
+
+## Interact with the generated project:
+Interactions can be done using HTTP requests either through `Curl`:
+```ssh
+$ curl -X POST "http://{host}:{port}/{application name}/contracts/helloworld/{contractAddress}/SayIt" -H  "accept: application/json" -H  "Content-Type: application/json" -d "{\"greeting\":\"Hello OpenAPI\"}"
+```
+Or, the `SwaggerUI`, on the link `{host}:{port}/swagger-ui`:
+
+![image](img/Web3j-OpenAPI/SwaggerUI_3.png)
+
+To interact via Java/Kotlin:
+
+```groovy
+dependencies {
+    implementation "web3j-openapi:web3j-openapi-client:4.6.4"
+}
+```
+
+Then, within a client application:
+
+```
+val service = ClientService("http://localhost:9090")
+val helloWorld = ClientFactory.create(HelloWorldProjectApi::class.java, service)
+
+println("Deploying the HelloWorld contract...")
+
+val receipt = helloWorld.contracts.helloWorld.deploy()
+
+println("Deployment receipt: ${receipt.contractAddress}")
```

Comment:
That would be "Deployed contract address: ${receipt.contractAddress}" to be more specific.